Environment_Engine/Query/Space.cs

```diff
@@ -44,7 +44,30 @@ public static partial class Query
         [Output("panelsAsSpace", "A collection of Environment Panels which have the given space name as a connected space")]
         public static List<Panel> ToSpace(this List<Panel> panels, string spaceName)
         {
-            return panels.Where(x => x.ConnectedSpaces.Contains(spaceName)).ToList();
+            List<Panel> result= panels.Where(x => x.ConnectedSpaces.Contains(spaceName)).ToList();
+            if(result.Count==0)
+                BH.Engine.Reflection.Compute.RecordWarning("Could not find following space: " + spaceName);
+            if(result.Count > 0 && result.Count<5)
+                BH.Engine.Reflection.Compute.RecordWarning("This space have less than 5 planar panels " + result.Count +" : "+ spaceName);
+
+            return result;
+
+        }
+
+        [Description("Returns a collection of Environment Panels which are connected to the list of space names provided")]
+        [Input("panels", "A collection of Environment Panels")]
+        [Input("spaceNames", " list of space names the panels should enclose")]
+        [Output("panelsAsSpace", "A collection of Environment Panels which have the given space name as a connected space")]
+        public static List<List<Panel>> ToSpace(this List<Panel> panels, List<string> spaceNames)
```

Comment:
Remove this method, there is no issue raised listed in this PR or elsewhere which describes the requirement for this method.

This method adds nothing that cannot be handled by the methods already existing within this file.